src/main/java/org/datadog/jenkins/plugins/datadog/DatadogGlobalConfiguration.java

```diff
@@ -1309,11 +1377,13 @@ public boolean isEmitSystemEvents() {
     @DataBoundSetter
     public void setEmitSystemEvents(boolean emitSystemEvents) {
         this.emitSystemEvents = emitSystemEvents;
+        this.createIncludeLists();
     }
 
     /**
      * @return - A {@link Boolean} indicating if the user has configured Datadog to emit Config Change events.
      */
+    @Deprecated
     public boolean isEmitConfigChangeEvents() {
```

Comment:
Do you know if we can remove these instead of deprecating them?

Reply:
Sure!